lib/evaluation/handlers/subjectIsHolderEvaluationHandler.ts

```diff
@@ -109,17 +108,17 @@ export class SubjectIsHolderEvaluationHandler extends AbstractEvaluationHandler
   }
 
   getAllInputDescriptorsWithAnyOfTheseFields(searchableFieldIds: Array<string>): Array<string> {
-    if (this.pDefinition && this.pDefinition.input_descriptors) {
+    if (this.pDefinition) {
```

Comment:
The previous code was more defensive as it checked pDefinition and input_descriptors to be present